### PR TITLE
Update servicing-stack-updates.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the document as we can now deploy Windows SSUs and LCUs together with one cumulative update

reference https://techcommunity.microsoft.com/t5/windows-it-pro-blog/deploy-windows-ssus-and-lcus-together-with-one-cumulative-update/ba-p/1967887

fixes https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10582

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/deployment/update/servicing-stack-updates.md [hunk 1]
```diff
@@ -40,7 +40,10 @@ Servicing stack update are released depending on new issues or vulnerabilities.
...
 
-Servicing stack updates must ship separately from the cumulative updates because they modify the component that installs Windows updates. The servicing stack is released separately because the servicing stack itself requires an update. For example, the cumulative update [KB4284880](https://support.microsoft.com/help/4284880/windows-10-update-kb4284880) requires the [May 17, 2018 servicing stack update](https://support.microsoft.com/help/4132216), which includes updates to Windows Update.
+Servicing stack updates improve the reliability of the update process to mitigate potential issues while installing the latest quality updates and feature updates. If you don't install the latest servicing stack update, there's a risk that your device can't be updated with the latest Microsoft security fixes.
+
+Beginning with the February 2021 LCU, microsoft will publish all future cumulative updates and SSUs for Windows 10, version 2004 and above together as one cumulative monthly update to the normal release category in WSUS.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/servicing-stack-updates.md:45]:
```suggestion
Beginning with the February 2021 LCU, Microsoft will publish all future cumulative updates and SSUs for Windows 10, version 2004 and later together as one cumulative monthly update to the normal release category in WSUS.
```

File: windows/deployment/update/servicing-stack-updates.md [hunk 2]
```diff
@@ -40,7 +40,10 @@ Servicing stack update are released depending on new issues or vulnerabilities.
...
+
```
Comment (JohanFreelancer9) [inline windows/deployment/update/servicing-stack-updates.md:46]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, you seem to have missed one suggested change. Could you also commit that, please?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@aczechowski Please merge this PR. Thanks

**Review by aczechowski — APPROVED** [review]

**Pull request merged**